### PR TITLE
Add custom reset to csv demo

### PR DESCRIPTION
**Pull request by benjamincharity**

### CODE REVIEW

File: projects/demo/src/app/components/csv-entry/csv-entry.component.ts [hunk 1]
```diff
@@ -1,5 +1,6 @@
-import { Component } from '@angular/core';
+import { Component, ViewChild } from '@angular/core';
```
Comment (atlwendy) [inline projects/demo/src/app/components/csv-entry/csv-entry.component.ts:1]: ..... just in demo though

File: projects/demo/src/app/components/csv-entry/csv-entry.component.ts [hunk 2]
```diff
@@ -17,6 +18,12 @@ export class CSVEntryComponent {
   public myFile;
   public footerDirection: 'ltr' | 'rtl' = 'ltr';
 
+  @ViewChild(TsCSVEntryComponent, {
+    static: true,
+    read: false,
```
Comment (atlwendy) [inline projects/demo/src/app/components/csv-entry/csv-entry.component.ts:23]: Haven't used this `read` option yet....

**Review by atlwendy — APPROVED** [review]

File: projects/demo/src/app/components/csv-entry/csv-entry.component.ts (hunk 1 above)
Comment (benjamincharity) [reply projects/demo/src/app/components/csv-entry/csv-entry.component.ts:1]: Yeah when we got rid of TSLint I relaxed some style-based rules in the demo and VR apps

File: projects/demo/src/app/components/csv-entry/csv-entry.component.ts (hunk 2 above)
Comment (benjamincharity) [reply projects/demo/src/app/components/csv-entry/csv-entry.component.ts:23]: I haven't really needed it before, but it's what hub is using and they are saying this doesn't work, so creating a reproduction they can see

**Pull request merged**